ansible/roles/candlepin/tasks/common/cp_debugging.yml

```diff
@@ -18,10 +18,13 @@
     - name: "Enable Tomcat debugging port in firewalld"
       become: true
       ansible.posix.firewalld:
-        port: "8000/tcp"
+        port: "{{ item }}/tcp"
         state: enabled
         permanent: true
         offline: false
+      with_items:
+        - 8000
```

Comment:
Let's add some comments on these so future maintainers don't need to look up port numbers to find it. We can use the same format we do in the base_deps task:

```
with_items:
  - 8000     # Tomcat remote debugging port
  - 5005     # Gradle remote debugging port
```

Reply:
Yeah, that is a good idea! Updated